snapshotTestContent/Authoring-MVP3/current_snapshots/snapshot_9000_0.json

```diff
@@ -1050,7 +1050,7 @@
         "path": "quorumValues",
         "type": "Blob",
         "value": {
-          "contents": "[[\"code2\",{\"approvalSequenceNumber\":3,\"commitSequenceNumber\":4,\"key\":\"code2\",\"sequenceNumber\":2,\"value\":\"@ms/office-fluid-container\"}],[\"leader\",{\"approvalSequenceNumber\":3837,\"commitSequenceNumber\":3838,\"key\":\"leader\",\"sequenceNumber\":3835,\"value\":\"c3931334-2f54-414d-932d-002cee6f4280\"}]]",
```

Comment:
can you also change code2 to code. rather get rid of that as nothing should be using it. it's very old

Reply:
You mean remove the first element of array which is "code2"?

Reply:
Isn't it the key-value pair?

Reply:
Would not the snapshot tests fails which matches the snapshots?

Reply:
i'm saying change all instance of `code2` to `code` just like you did by making the value and object

Reply:
Done. I got confused by when you said get rid of it. :)